packages/matrix/docker/synapse/index.ts

```diff
@@ -307,3 +309,48 @@ export async function createRegistrationToken(
     );
   }
 }
+
+export async function updateUser(
+  synapse: SynapseInstance,
+  adminAccessToken: string,
+  userId: string,
+  {
+    password,
+    displayname,
+    avatar_url,
+    emailAddresses,
+  }: {
+    password?: string;
+    displayname?: string;
+    avatar_url?: string;
+    emailAddresses?: string[];
+  },
+) {
+  let res = await fetch(
+    `http://localhost:${synapse.mappedPort}/_synapse/admin/v2/users/${userId}`,
```

Comment:
Any reason we use localhost here but 127.0.0.1 on line 240 above?

Reply:
no reason, i think it was just copy-pasta